chapter_3/CMakeLists.txt

```diff
@@ -11,7 +11,9 @@ include_directories("${PROJECT_SOURCE_DIR}/libflandmark")
 FILE(COPY "haarcascade_frontalface_alt.xml" DESTINATION ${CMAKE_BINARY_DIR})
 FILE(COPY "flandmark_model.dat" DESTINATION ${CMAKE_BINARY_DIR})
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+set(CMAKE_C_COMPILER "/usr/bin/clang")
+set(CMAKE_CXX_COMPILER "/usr/bin/clang++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++ -stdlib=libstdc++")
```

Comment:
I am wondering why you need to tell you compiler to use C++11 explicitly. This is an issue on your end, because I am running older C++ compilers here without any issues. Again, this is something the user should manage themselves but this is not a global solution